src/integration/billing/BillingIntegration.ts

```diff
@@ -364,6 +366,28 @@ export default abstract class BillingIntegration<T extends BillingSetting> {
         detailedMessages: { error: error.message, stack: error.stack }
       });
     }
+    const tenant = await TenantStorage.getTenant(this.tenantID);
+    // Send async notification
+    NotificationHandler.sendBillingNewInvoiceNotification(
+      this.tenantID,
+      Utils.generateGUID(),
+      invoice.user,
+      {
+        user: invoice.user,
+        evseDashboardInvoiceURL: await Utils.buildEvseBillingInvoicesURL(this.tenantID),
+        evseDashboardURL: Utils.buildEvseURL(tenant.subdomain),
+        invoiceDownloadUrl: await Utils.buildEvseBillingDownloadInvoicesURL(this.tenantID, invoice.id),
+        invoice: invoice
+      }
+    ).catch((error) => {
```

Comment:
Don't use this form of catch here, this is an async call and we don't want to handle errors here.

Reply:
Removed